src/Features/Core/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.GlobalSuppressMessageCodeAction.cs

```diff
@@ -41,13 +41,8 @@ public GlobalSuppressMessageCodeAction(AbstractSuppressionCodeFixProvider fixer,
                 };
             }
 
-            public override string Title
-            {
-                get
-                {
-                    return _title;
-                }
-            }
+            public override string Title => _title;
+            public override string EquivalenceKey => _title + _diagnostic.Id + _targetSymbol.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat);
```

Comment:
Should we use the documentationId instead of the displaystring since it's the documentationid that gets spit into the attribute to represent the targetsymbol?